compiler/visq/visq

```diff
@@ -107,6 +117,66 @@ def _run_dalgona(model, data, analysis, save_dir):
         raise
 
 
+# Generate h5 file that contains a dataset of a single batch
+# This is for batch execution of visq
+def gen_batch_h5(inputs_data, inputs_path):
+    # Create h5 file
+    output_path = inputs_path + "/inputs.h5"
+    h5_file = h5.File(output_path, 'w')
+    group = h5_file.create_group("value")
+    group.attrs['desc'] = "Input data"
+
+    for i in range(len(inputs_data)):
+        sample = group.create_group(str(i))
+        for j in range(len(inputs_data[i])):
+            sample.create_dataset(str(j), data=inputs_data[i][j])
+
+    h5_file.close()
+    return output_path
+
+
+# Aggregate intermediate results for a given data
+def advance_on_data(fp32_model, fq_model, data, computers):
+
+    curr_dir = Path(__file__).parent.resolve()
+    dump_fp32_py = curr_dir / 'visqlib' / 'DumpFP32FM.py'
+    dump_fq_py = curr_dir / 'visqlib' / 'DumpFakeQuantFM.py'
+
+    with tempfile.TemporaryDirectory() as fp32_dir, \
+         tempfile.TemporaryDirectory() as fq_dir:
+
+        #print("advance on data + {fp_dir} + {fq_dir}".format(fp_dir = fp32_dir, fq_dir = fq_dir))
```

Comment:
Q) what is the need for this comment?

Reply:
Sorry. No. We don't need it. I'll remove it.